pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/DefaultRebalancePreChecker.java

```diff
@@ -74,16 +74,17 @@ public Map<String, RebalancePreCheckerResult> check(PreCheckContext preCheckCont
     String tableNameWithType = preCheckContext.getTableNameWithType();
     TableConfig tableConfig = preCheckContext.getTableConfig();
     RebalanceConfig rebalanceConfig = preCheckContext.getRebalanceConfig();
+    Logger tableRebalanceLogger = LoggerFactory.getLogger(LOGGER.getName() + "_" + rebalanceJobId);
```

Comment:
Consider also including table name, then we can simplify the log content to not log table name
```suggestion
    Logger tableRebalanceLogger = LoggerFactory.getLogger(getClass().getSimpleName() + "-" + tableNameWithType + "-" rebalanceJobId);
```

Reply:
If we want to unify the logger for these 2 classes, we may also name the logger `Rebalance-<tableNameWithType>-<jobId>`

Reply:
> If we want to unify the logger for these 2 classes, we may also name the logger Rebalance-<tableNameWithType>-<jobId>

I think it's okay to keep them separate. @somandal any thoughts?

Reply:
I don't have a strong preference, keeping separate might be good if we want to look at specific logs for pre-check vs. rebalance